### PR TITLE
Update the bootstrapper to install the compiled version of KoreBuild

### PR DESCRIPTION
**Pull request by natemcmaster**
- Replace SplitPackages as a package reference with a RepoTask instead
 - Replace build.ps1/sh with the default bootstrapper for compiled korebuild

### CODE REVIEW

File: build.ps1 [hunk 1]
```diff
@@ -1,68 +1,177 @@
...
-            $exceptionMessage = $_.Exception.Message
-            Write-Host "Failed to download '$url': $exceptionMessage"
-            if ($retries -gt 0) {
-                $retries--
-                Write-Host "Waiting 10 seconds before retrying. Retries left: $retries"
-                Start-Sleep -Seconds 10
+#!/usr/bin/env powershell
+#requires -version 4
+
+<#
```
Comment (ryanbrandenburg) [inline build.ps1:4]: I assume these structured comments end up used in Intelisense or error messages under the correct conditions?

File: build.ps1 [hunk 2]
```diff
@@ -1,68 +1,177 @@
...
+    [Alias('c')]
+    [string]$Channel,
+    [Alias('d')]
+    [string]$DotNetHome,
+    [Alias('s')]
+    [string]$ToolsSource,
+    [Alias('u')]
+    [switch]$Update,
+    [string]$ConfigFile = (Join-Path $PSScriptRoot 'version.xml'),
+    [Parameter(ValueFromRemainingArguments = $true)]
```
Comment (ryanbrandenburg) [inline build.ps1:62]: This is kind of like *args in python? A catch all for anything that's left?

File: build/tasks/RepoTasks.csproj [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (ryanbrandenburg) [inline build/tasks/RepoTasks.csproj:1]: I believe RepoTasks was your idea @natemcmaster? I like it.

**Review by ryanbrandenburg — APPROVED** [review]
Looks solid, excited to get this rolling.

File: build/tasks/RepoTasks.csproj (hunk 3 above)
Comment (natemcmaster) [reply build/tasks/RepoTasks.csproj:1]: 👍 repo tasks is ideal for "tools" that are only used in one place. Like SplitPackages. It doesn't really need to be a console tool.

File: build.ps1 (hunk 2 above)
Comment (natemcmaster) [reply build.ps1:62]: Yup.

File: build.ps1 (hunk 1 above)
Comment (natemcmaster) [reply build.ps1:4]: Yup. Try this in powershell: `get-help build.ps1`

**Pull request merged**